tools/script_template.py

```diff
@@ -1,5 +1,43 @@
 import argparse
 import yaml
+import alara_output_processing as aop
+import openmc
+import numpy as np
+
+def write_out_tables(run_out, run_lbl, time_unit):
+    '''
+    Takes the output of a single ALARA run and converts output tables to Pandas DataFrame
+    inputs: 
+        run_out (path to ALARA output file)
+        run_lbl (name of the run)
+        time_unit (units for cooling time)
+    output: output_tables (Pandas DataFrame containing all output data)
+    '''
+    alara_data = aop.FileParser(run_out, run_lbl, time_unit)
+    output_tables = alara_data.extract_tables()
+    return output_tables
+
+def open_flux_file(flux_file):
+    flux_lines = open(flux_file, 'r').readlines()
+    return flux_lines
+
+def parse_flux_lines(flux_lines):
+    '''
+    Uses provided list of flux lines and group structure applied to the run to create an array of flux entries, with:
+    # rows = # flux entries / # group structure bins
+    # columns = # group structure bins
+    input : flux_lines (list of lines from ALARA flux file)
+    output : flux_array (numpy array of shape # zones/intervals/mixtures x )
+    '''
+    energy_bins = openmc.mgxs.GROUP_STRUCTURES['VITAMIN-J-175']     
+    bin_widths = energy_bins[1:] - energy_bins[:-1]
+    all_entries = []
+    for flux_line in flux_lines:
+        if flux_line.strip(): #if the current line is not blank
+            all_entries.extend(flux_line.split())
```

Comment:
I wonder if it's possible to do something like:
```
all_entries = ' '.join(flux_lines).split()
```
This should just join all the lines together, adding an extra `' '` at each new line to be safe, and then split based on whitespace, thus elimating all the extra spaces and new lines. 

I haven't tested it, but it seems like it should work.